reference/tools/user-guide.md

```diff
@@ -0,0 +1,187 @@
+- Data export tools, including full export tools. incremental export tools, and so forth.
+
+The two types of tools are discussed in some details below.
+
+### Data import tools
+
+#### Full import tool loader (Stop maintenance, not recommended)
+
+[Loader](/reference/tools/loader.md) is a lightweight full data import tool. Data in the form of SQL is imported into TiDB. Currently this tool is gradually replaced by [TiDB Lightning](#full-import-tool-tidb-lightning), see [TiDB Lightning TiDB-backend Document](/reference/tools/tidb-lightning/tidb-backend.md#migrating-from-loader-to-tidb-lightning-tidb-backend).
+
+The following is the basics of Loader:
+
+- Input: Files output on the Mydumper.
+- Output: SQL written to TiDB.
+- Applicable version of TiDB: All.
+- Kubernetes: Supported. See [Backup and restore](/tidb-in-kubernetes/maintain/backup-and-restore.md).
+
+#### Full import tool TiDB Lightning
+
+[TiDB Lightning](/reference/tools/tidb-lightning/overview.md) is a tool used for fast full import of data into a TiDB cluster.
+
+> **Note:**
+>
+> When you import data into TiDB using TiDB Lightning, there are two modes:
+>
+> - The default mode: Use `tikv-importer` as the backend. In this mode, the cluster can not provide normal services during the data import process. It is used when you import large amounts (TBs) of data.
+> - The second mode: Use `TiDB` as the backend (similar to Loader). The  import speed is slower than that of the default mode. However, the second mode supports online import.
+
+The following is the basics of TiDB Lightning:
+
+- Input:
+    - Files output on the Mydumper;
+    - CSV-formatted files.
+- Applicable version of TiDB: v2.1 and later versions.
+- Kubernetes: Supported. See [Quick restore data into a TiDB cluster in Kubernetes using TiDB Lightning](/tidb-in-kubernetes/maintain/lightning.md).
+
+#### Backup and restore tool BR
+
+[BR](/how-to/maintain/backup-and-restore/br.md) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with Mydumper and Loader, BR is more efficient, and more suitable for scenarios of huge data volume.
+
+The following is the basics of BR:
+
+- [Types of backup files](/how-to/maintain/backup-and-restore/br.md#types-of-backup-files): The SST file and the `backupmeta` file.
+- Applicable version of TiDB: v3.1 and v4.0.
+- Kubernetes: Supported. Relevant documents are on the way.
+
+#### Incremental import tool Syncer (Stop maintenance, not recommended)
+
+[Syncer](/reference/tools/syncer.md) is a tool used for incremental import of real-time binlog data from MySQL/MariaDB into TiDB. It is recommended to use [TiDB Data Migration](#Incremental-import-tool-tidb-data-migration) to replace Syncer.
+
+The following is the basics of Syncer:
+
+- Input: Binlog data of MySQL/MariaDB.
+- Output: SQL written to TiDB.
+- Applicable version of TiDB: All.
+- Kubernetes: Not supported.
+
+#### Incremental import tool TiDB Data Migration
```

Comment:
DM support incremental and full data's export and import.

Reply:
Thanks a lot.